subsys/power/Kconfig

```diff
@@ -8,39 +8,16 @@ config PM_CONTROL_OS_LPS
 	help
 	  Select this option if SoC support LPS state.
 
-if PM_CONTROL_OS_LPS
-config PM_LPS_MIN_RES
-	int "LPS minimum residency"
-	default 5
-	help
-	  Minimum residency in ticks to enter LPS state.
-endif
-
 config PM_CONTROL_OS_LPS_1
 	bool "Platform supports LPS_1"
 	help
 	  Select this option if SoC support LPS_1 state.
```

Comment:
Side comment: 

For me the naming of states creates some confusion. I really do now know how many states are in fact supported on my platform and how Zephyr names maps to platform capabilities. 

IMHO each platform which supports any kind of power management should define own list of supported states (Example: my platform support LPS_1, LPS_2 and DEEP_SLEEP) and provide some information how they map to the names in datasheet. User should be able to disable some states in Kconfig, however we should not force him to solve multi-selection test about power management.

Reply:
@pizi-nordic  I think we can add the individual supported states as Kconfig option at arch level like SYS_POWER_LOW_POWER_STATE_SUPPORTED and SoC implementations can select them based on the SoC specific support.

So, in policy Kconfig we make them depend on the arch level Kconfig, that way we will only have what is supported by a SoC.

Reply:
I think that it is a very good idea.